service/matching/fx.go

```diff
@@ -177,38 +177,6 @@ func VisibilityManagerProvider(
 	)
 }
 
-func HandlerProvider(
```

Comment:
Is this change coming from a different branch?

Reply:
This is in main. Not sure why it's showing up in this diff